mitosheet/src/mito/utils/actions.tsx

```diff
@@ -321,6 +322,28 @@ export const getActions = (
                 windows: 'Ctrl+C'
             }
         },
+        [ActionEnum.CopyCode]: {
+            type: 'build-time',
+            staticType: ActionEnum.CopyCode,
+            icon: CopyIcon,
+            toolbarTitle: 'Copy Code to Clipboard',
+            longTitle: 'Copy Code to Clipboard',
+            actionFunction: () => {
+                closeOpenEditingPopups();
+
+                const stringToCopy = analysisData.code.join('\n');
+
+                void writeTextToClipboard(stringToCopy);
+            },
+            isDisabled: () => {
+                if (!doesAnySheetExist(sheetDataArray)) {
+                    return 'There is no code to copy. Import data.';
+                }
```

Comment:
Is this the right disabled? I think there could still be code in this case. I think we want to check that analysisData.code.length > 0